app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,26 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title Target
+    = csrf_meta_tags
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{:href => "//fonts.googleapis.com/css?family=Open+Sans:light,bold,italic,800,700,300,600,800,400", :rel => "stylesheet", :type => "text/css"}/
```

Comment:
Please use newer syntax href: "something"
In a new pr please add code quality tools (rubocop, brakeman, reek, rails_best_practices). Let me know if you need any help to do it